lua/cfc_random_spawn/module/sv_player_spawning.lua

```diff
@@ -1,70 +1,79 @@
+cfcRandomSpawn = cfcRandomSpawn or {}
+
 local customSpawnsForMap = cfcRandomSpawn.config.CUSTOM_SPAWNS[game.GetMap()]
 local mapHasCustomSpawns = customSpawnsForMap ~= nil
 
 hook.Remove( "PlayerSpawn", "CFC_PlayerSpawning" )
 
-if mapHasCustomSpawns then
-    cfcRandomSpawn.spawnPointRankings = cfcRandomSpawn.spawnPointRankings or {}
+if not mapHasCustomSpawns then return end
 
-    local function getMeasurablePlayers()
-        local measurablePlayers = {}
-        for _, ply in pairs( player.GetHumans() ) do
-            if ( ply:Alive() ) then
-                table.insert( measurablePlayers, ply )
-            end
-        end
+cfcRandomSpawn.spawnPointRankings = cfcRandomSpawn.spawnPointRankings or {}
 
-        return measurablePlayers
+local function getMeasurablePlayers()

```

Comment:
We could do something stupid here, and maintain a cache of alive players and only invalidate it when a player dies or spawns - that way it'd be _slightly_ more efficient